Crashes if either Enzyme or RTL are not a project dependency

I tried this out on a repo that only has RTL as a dependency:

```ts
import { setupRtl } from 'component-test-setup';
```

```
  ● Test suite failed to run

    Cannot find module 'enzyme' from 'node_modules/component-test-setup/src/setupEnzyme.js'

    Require stack:
      node_modules/component-test-setup/src/setupEnzyme.js
      node_modules/component-test-setup/src/index.js
      src/components/MyComponent.test.tsx

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (node_modules/component-test-setup/src/setupEnzyme.js:7:18)
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -14,5 +14,6 @@ module.exports = {
   rules: {
     "@typescript-eslint/explicit-module-boundary-types": "off",
     "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-var-requires": "off",
   },
 };

--- a/package.json
+++ b/package.json
@@ -60,5 +60,5 @@
     "tsc": "tsc --module commonjs"
   },
   "types": "src/index.d.ts",
-  "version": "0.2.5"
+  "version": "0.3.0-alpha"
 }

--- a/package.json
+++ b/package.json
@@ -60,5 +60,5 @@
     "tsc": "tsc --module commonjs"
   },
   "types": "src/index.d.ts",
-  "version": "0.3.0-alpha"
+  "version": "0.3.0-beta"
 }

--- a/src/setupEnzyme.tsx
+++ b/src/setupEnzyme.tsx
@@ -1,4 +1,3 @@
-import { mount } from "enzyme";
 import React from "react";
 
 import {
@@ -30,6 +29,7 @@ export function setupEnzyme<
   /* eslint-disable-next-line @typescript-eslint/ban-types */
   BaseProps extends Partial<FullProps<ComponentType>> = {}
 >(Component: ComponentType, baseProps?: BaseProps): RenderEnzyme<ComponentType, BaseProps> {
+  const { mount } = require("enzyme") as typeof import("enzyme");
   return function renderWrapper(
     testProps?: RemainingPropsAndTestOverrides<ComponentType, BaseProps>,
   ) {

--- a/src/setupRtl.tsx
+++ b/src/setupRtl.tsx
@@ -1,4 +1,4 @@
-import { render, RenderOptions } from "@testing-library/react";
+import type { RenderOptions } from "@testing-library/react";
 import React from "react";
 
 import { RemainingPropsAndTestOverrides, FullProps, RenderRtl, SetupComponentType } from "./types";
@@ -25,6 +25,7 @@ export function setupRtl<
   /* eslint-disable-next-line @typescript-eslint/ban-types */
   BaseProps extends Partial<FullProps<ComponentType>> = {}
 >(Component: ComponentType, baseProps?: BaseProps): RenderRtl<ComponentType, BaseProps> {
+  const { render } = require("@testing-library/react") as typeof import("@testing-library/react");
   let options: RenderOptions;
 
   function renderView(testProps?: RemainingPropsAndTestOverrides<ComponentType, BaseProps>) {

